package.json

```diff
@@ -108,6 +108,7 @@
     ]
   },
   "dependencies": {
+    "@bramus/specificity": "github:bramus/specificity",
```

Comment:
[note] The head version is needed but not released yet:
https://github.com/bramus/specificity/commit/09b42a46dce1928947af94a13b2b17deaf64394a